config/routes.rb

```diff
@@ -5,4 +5,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+  end
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added,can%20see%20the%20following%20output%3A) to Make sure that you add routes to handle only the 4 required URLs.
* currently all the unnecessary endpoints as being exposed, Kindly limit your routes to only actions needed in your controllers For example 👇 

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```